src/python/nimbusml/base_predictor.py

```diff
@@ -39,16 +38,6 @@ def fit(self, X, y=None, **params):
         :param y: array-like with shape=[n_samples]
         :return: self
         """
-        if y is not None and not isinstance(y, (
-                str, tuple)) and self.type in set(
-                ['classifier', 'anomaly']):
-            unique_classes = unique_labels(y)
-            if len(unique_classes) < 2:
-                raise ValueError(
-                    "Classifier can't train when only one class is "
-                    "present.")
-            self.classes_ = unique_classes
```

Comment:
>self.classes_  [](start = 12, length = 14)

So classes_ is not set on predictor level now?

Reply:
It is still set on a `BasePredictor` but the assignment has moved in to the `Pipeline._extract_classes()` and `Pipeline._add_classes()` method.